src/components/liveSession/components/liveSessionBtn/liveSessionBtn.component.js

```diff
@@ -39,11 +48,18 @@
                 }
 
                 function showSessionModal() {
-                    $mdDialog.show({
-                        template: '<live-session-subject-modal student="vm.student"></live-session-subject-modal>',
-                        scope: $scope,
-                        preserveScope: true,
-                        clickOutsideToClose: true
+                    getScheduledLesson().then(scheduledLesson => {
+                        if (scheduledLesson) {
+                            vm.lessonId = scheduledLesson.id;
+                            $mdDialog.show({
+                                template: '<live-session-subject-modal student="vm.student" lesson-id="vm.lessonId"></live-session-subject-modal>',
+                                scope: $scope,
+                                preserveScope: true,
+                                clickOutsideToClose: true
+                            });
+                        } else {
```

Comment:
Currently - if there's no lesson scheduled then after the teacher clicks "Start Session" => nothing will happen.
It would look broken.

Add a popup with the message:
"Cannot start lesson,
there is no lesson scheduled with [student name] at this time."

Reply:
done